### ISSUE TITLE
Alliance trauma quest for First aid is not available

### ISSUE THREAD

**Issue by templar444** [slowtorta/turtlewow-bug-tracker#1848]
On level 225 of First Aid profession both faction players should complete a small quest chain. For alliance it starts with Nissa Firestone at Ironforge. The quest is visible on minimap, but the NPC doesn't have it. Can't develop First Aid further because of it.

NPC https://database.turtle-wow.org/?npc=5150
Quest https://database.turtle-wow.org/?quest=6625

**Comment by eX1stenZT** [comment]
I did get the quest available at 225 (as soon as I skilled up in front of him), however after accepting I got a quest log full error. He's unable to interact with anyone after it. 2 people tried, he won't open dialogue.
So not quite sure if perhaps the quest log full error could somehow break him (would have thought that'd be a handled error/exception along quest interfaces, but dunno wow's internals).
